lib/query-interface.js

```diff
@@ -189,12 +189,20 @@ class QueryInterface {
       return Promise.all(promises).then(results => {
         const promises = [];
         let enumIdx = 0;
+        let type;
 
         for (i = 0; i < keyLen; i++) {
-          if (attributes[keys[i]].type instanceof DataTypes.ENUM) {
+          type = attributes[keys[i]].type;
+          // Handle arrays of enum type.
+          if (type instanceof DataTypes.ARRAY) type = type.type;
+
+          const attr = attributes[keys[i]].field || keys[i];
+          if (type instanceof DataTypes.ENUM) {
+            type._enumName = this.QueryGenerator.pgEnumName(tableName, attr, { schema: false });
```

Comment:
Why you need to set this, `_enumName` used anywhere else or it should be `enumName` ?

Reply:
Good catch, this isn't needed to be set, will clean that up.

Reply:
Was left over code from a previous approach I had tried that I forgot to remove.